.github/workflows/checks.yml

```
@@ -84,5 +84,37 @@ jobs:
       - name: Install the project
         run: uv sync --frozen --all-extras --dev
 
+      - name: Start Inspector test server
+        run: |
+          # Create a minimal test script to start the inspector
+          cat > test_inspector_server.py << 'EOF'
+          import time
+          from mcp_agent.inspector import mount
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)